File: client-phase10/src/app/player/playerResourceModel.ts
import { Card } from '../card/card';
import { Phase, PhaseType } from '../phase/phase';
import { Stage } from '../game/stage';

export interface PlayerResourceModel{
    id: string;
    name: string;
    hand: Card[];
    phase: Phase;
    stage: Stage;
    hasCompletedPhase: boolean;
}

File: server/Models/Dtos/PlayerResourceModel.cs
using System.Collections.Generic;

namespace server.Models.Dtos
{
    public class PlayerResourceModel
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public List<CardResourceModel> Hand { get; set; }
        public PhaseResourceModel Phase { get; set; }
        public int Stage { get; set; }
        public bool HasCompletedPhase { get; set; }
    }
}